ipl.py

```diff
@@ -0,0 +1,67 @@
+from cricbuzz import *
+from time import sleep
+import tags
+
+
+seriesName = "Indian Premier League, 2017"
+
+currentBall = "0.0" #to track change in balls/over
+currentMatchStatus = "" #to track change in match status, if raining or whatever
+
+
+
+def ballChanged(match_detail):
+	global currentBall
+	if match_detail[tags.BattingTeamOvers] != currentBall:
+		currentBall = match_detail[tags.BattingTeamOvers]
+		return True
+	else :
+		return False	
+
+def printProgressIfChanged(match_detail):
+	global currentMatchStatus
+	if currentMatchStatus == "":
+		currentMatchStatus = match_detail[tags.MatchState]
+		return 
+	if match_detail[tags.MatchState] != currentMatchStatus:
+		currentMatchStatus = match_detail[tags.MatchState]
+		print "Match Status Changed, "+match_detail[tags.MatchState]
+
+
+def printIplDetails():
```

Comment:
If you are trying to give example of using `cricbuzz` library, I would suggest you to create a directory named `examples` and put the code inside it.